packages/imperative/src/profiles/src/abstract/AbstractProfileManager.ts

```diff
@@ -233,7 +233,7 @@ export abstract class AbstractProfileManager<T extends IProfileTypeConfiguration
             this.validateConfigurationDocument(config);
         }
         this.mProfileTypeSchema = this.mProfileTypeConfiguration.schema;
-        this.mProfileTypeRootDirectory = this.createProfileTypeDirectory();
+        this.mProfileTypeRootDirectory = this.profileRootDirectory + "/" + this.profileType + "/";
```

Comment:
Very important change here.
We still provide protected getter `profileTypeRootDirectory` but at least we are not creating the directory anymore 😋 

I guess we still need the getter for conversion purposes 😋